lib/spotlight/dor/indexer.rb

```diff
@@ -31,6 +31,10 @@ def solr_document(resource)
         before_index :add_author_no_collector
         before_index :add_collector
         before_index :add_genre
+        before_index :add_box
+        before_index :add_folder
+        before_index :add_location
+        before_index :add_series
```

Comment:
and again, sorry if I was unclear:  I meant a comment indicating the physical location ones are grouped together  or a single alpha sort including genre, collector, etc.